docs/common_links.txt

```diff
@@ -108,8 +108,8 @@
 .. _`scipy mailing list`: http://mail.python.org/mailman/listinfo/scipy-dev
 .. |minimum_scipy_version| replace:: {scipy}
 
-.. asdf
-.. |minimum_asdf_version| replace:: {asdf}
+.. asdf-astropy
+.. |minimum_asdf_astropy_version| replace:: {asdf_astropy}
```

Comment:
I actually don't know if this should be `{asdf_astropy}` or `{asdf-astropy}`. I guess I'll see if the doc build fails.